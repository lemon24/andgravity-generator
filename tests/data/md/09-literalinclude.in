.. literalinclude:: file

.. literalinclude:: file.py

.. literalinclude:: file.py
    :linenos:

.. literalinclude:: file.py
    :linenos:
    :lines: 2-3

.. literalinclude:: file
    :lines: 1,3,5-
    :linenos: no
    :ellipsis: ...
