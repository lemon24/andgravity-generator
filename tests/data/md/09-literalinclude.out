<pre class="code code-container"><code>one
two
three
four
five

</code></pre>

<div class="highlight code-container"><pre class="code" data-lang="Python"><span></span><code><span class="n">one</span>
<span class="n">two</span>
<span class="n">three</span>
<span class="n">four</span>
<span class="n">five</span>

</code></pre></div>

<div class="highlight code-container"><table class="highlighttable"><tr><td class="linenos"><div class="linenodiv"><pre class="code"><code><span class="normal">1</span>
<span class="normal">2</span>
<span class="normal">3</span>
<span class="normal">4</span>
<span class="normal">5</span>
<span class="normal">6</span></code></pre></div></td><td class="code"><div><pre class="code" data-lang="Python"><span></span><code><span class="n">one</span>
<span class="n">two</span>
<span class="n">three</span>
<span class="n">four</span>
<span class="n">five</span>

</code></pre></div></td></tr></table></div>

<div class="highlight code-container"><table class="highlighttable"><tr><td class="linenos"><div class="linenodiv"><pre class="code"><code><span class="normal">2</span>
<span class="normal">3</span></code></pre></div></td><td class="code"><div><pre class="code" data-lang="Python"><span></span><code><span class="n">two</span>
<span class="n">three</span>
</code></pre></div></td></tr></table></div>

<pre class="code code-container"><code>one
...
three
...
five

</code></pre>
